www/docs/adapters/prisma.md

```diff
@@ -23,7 +23,7 @@ Configure your NextAuth.js to use the Prisma Adapter:
 ```javascript title="pages/api/auth/[...nextauth].js"
 import NextAuth from "next-auth"
 import Providers from "next-auth/providers"
-import { PrismaAdapter } from "@next-auth/prisma-adapter"
+import PrismaAdapter from "@next-auth/prisma-adapter"
```

Comment:
In the [source code of the adapters repo](https://github.com/nextauthjs/adapters/blob/canary/packages/prisma/src/index.ts#L6), this is a named export. So I think the docs are correct 💭

Reply:
Yes, @lluia is right. If you experience otherwise, please open a bug report and attach a reproduction.

Reply:
Ah, I had the wrong version installed, thanks for the info and sorry for the spam PR